index.js

```diff
@@ -0,0 +1,124 @@
+$(document).ready(() => {
+  const baseUrl = "https://trektravel.herokuapp.com";
+
+  const travelList = function travelList() {
+    const listPartial = "/trips";
+    if (".individual-trip-section") {
+      $(".individual-trip-section").remove();
+    }
+    if ("#ajax-results p") {
+      $("#ajax-results p").remove();
+    }
+    $.get(baseUrl + listPartial, response => {
+      $(".list-trips").html(
+        `<div class = "removable-trip-list  row small-up-2 medium-up-2 large-up-3 "></div>`
+      );
+      for (let trip of response) {
+        $(".removable-trip-list").append(
+          `<div class = "container column column-block "><h3 class="trip" > ${
+            trip["name"]
+          }</h3> <div class="overlay"  id ="${trip["id"]}">
+    <div class="text">Learn More</div>
+  </div><div>`
+        );
+      }
+    })
+      .fail(function(response) {
+        console.log(response);
+        $("#fail").html("<p>Request was unsuccessful</p>");
+      })
+      .always(function() {
+        console.log("always even if we have success or failure");
+      });
+  }; //end travelList func
+
+  const singleTrip = function singleTrip(number) {
+    const singleTripUrlPartial = "/trips/";
+    if (".removable-trip-list") {
+      $(".removable-trip-list").remove();
+    }
+    $.get(baseUrl + singleTripUrlPartial + number, response => {
+      // let wonderAddress = response.results[0].formatted_address;
+      console.log(response);
+      $(".individual-trip").html(
+        `<section class = "individual-trip-section row small-up-1 medium-up-1 large-up-2"><div class = "column column-block"> <h2>${
+          response["name"]
+        }</h2> <button class = 'hollow button' id="reserve">Reserve Trip <i class="fa fa-plane"></i></button> <h4>Trip Details</h4> <ul><li class = 'trip-id' id='${
+          response["id"]
+        }'>Id: ${response["id"]} </li> <li>Category: ${
+          response["category"]
+        }</li> <li>Continent: ${
+          response["continent"]
+        }</li> <li>Duration(weeks): ${
+          response["weeks"]
+        }</li> <li>Price(USD): $${parseFloat(response["cost"]).toFixed(
+          2
+        )}</li> </ul></div> <form class = "column column-block" id="add-reservation-form" action="index.html" method="post"></form><div class = "description"> <h4>Description</h4> <p> ${
+          response["about"]
+        }</p></div></section>`
+      );
+      console.log(response);
+    }).fail(function(response) {
+      $("#fail").html("<p>Request was unsuccessful</p>");
+    });
+  };
+  const callback = function response() {
+    console.log("successful");
+    console.log(response);
+    let generatedHTML = "<p>Your reservation for trip has been added</p>";
```

Comment:
"Callback" is not a particularly descriptive name for this function - it's almost like calling it "function". When is this callback function run? What does it do? These should guide your naming.

Something like "tripReservationSuccess" might be a good name.

Also, with an AJAX success/failure callback, `response` should be an argument. I'm guessing that line 67 would always print out `unknown`.